spot-contracts/contracts/_interfaces/IBondIssuer.sol

```diff
@@ -8,10 +8,17 @@ interface IBondIssuer {
     /// @param bond The newly issued bond.
     event BondIssued(IBondController bond);
 
+    /// @notice Event emitted when a bond has matured.
+    /// @param bond The mature bond.
+    event BondMature(IBondController bond);
+
     /// @notice The address of the underlying collateral token to be used for issued bonds.
     /// @return Address of the collateral token.
     function collateral() external view returns (address);
 
+    /// @notice Invokes `mature` on issued bonds.
```

Comment:
Should we comment that it fails if there are no mature operations to execute? This would be behavior that clients like Gelato would rely on.

One step further would be to name the function `matureAllOrRevert()`

Reply:
Seems like `matureActive()` is more appropriate. I think its reasonable to assume that it reverts if no active bonds exist. Theres a `@dev` comment on above the function implementation about the revert ..